prep-javascript-for-loops/main.js

```diff
@@ -0,0 +1,4 @@
+// create your loops here.
+for (let i = 100; i > 0; i--) {
+  console.log('Time till explosion: ' + i + '!');
+}
```

Comment:
Hi Daniel, please include all three loops in this file.

Reply:
<img width="555" alt="Screen Shot 2022-03-03 at 2 58 34 PM" src="https://user-images.githubusercontent.com/20695764/156667073-abc9bdb8-cba0-4205-b789-ac34ab0f8602.png">